recipes/bigdft/test.py

```diff
@@ -0,0 +1,19 @@
+from BigDFT.Systems import System
+from BigDFT.Fragments import Fragment
+from BigDFT.Atoms import Atom
+
+at = Atom({"He": [0, 0, 0]})
+frag = Fragment([at])
+sys = System({"FRA:0": frag})
+
+from BigDFT.Calculators import SystemCalculator
+code = SystemCalculator()
+
+from BigDFT.Inputfiles import Inputfile
+inp = Inputfile()
+inp.set_xc("PBE")
+inp.set_hgrid(0.4)
+
+log = code.run(sys=sys, input=inp)
+print(log.energy)
+
```

Comment:
```suggestion

import CheSS
import futile
import gi
```
Test imports of all your packages.

Reply:
CheSS is depreciated (I'm put in a pull request to remove it from our main repository) and gi is too experimental to test, but testing the imports explicitly of futile and BigDFT sounds like a good idea to me (better than the error coming from deep in a run). I've made a modification based on this suggestion.

Reply:
The tests in this conda recipe are for *packaging* validation, so their purpose is to check that the artifacts were installed correctly. They are not necessarily a comment on whether a feature of the package is deprecated or considered experimental. If you expect that an end user of this package would still be importing the CheSS module, then you should test it.